### ISSUE TITLE
Correct KIKI-net

### ISSUE THREAD

**Issue by zaccharieramzi** [zaccharieramzi/fastmri-reproducible-benchmark#9]
Currently the implemented kiki-net is actually ikik. Shouldn't make much of difference.

However, the training is supposed to be iterative, i.e. the i and k nets should be trained separately.

Moreover, to avoid dying ReLU, we should use LeakyReLU rather than ReLU (this is probably why the net is so bad at this point after some trials on a small part of the training set).

Weight decay is also important to fix the overfitting (I immediately saw it when training the first K-net), however, a first good step would just be to add L2 weight regularisation and in a second step use [AdamWOptimizer](https://www.tensorflow.org/api_docs/python/tf/contrib/opt/AdamWOptimizer) that implements the weight decay correction noted [in this paper](https://arxiv.org/abs/1711.05101).

**Comment by zaccharieramzi** [comment]
Actually, [this paper](http://proceedings.mlr.press/v102/souza19a/souza19a.pdf) claims that the separate training is not necessary to ensure proper convergence.

Still keeping it as a possibility in the PR to come.
